desc/profiles.py

```diff
@@ -335,6 +346,128 @@ def __repr__(self):
         return s
 
 
+class PoweredProfile(_Profile):
```

Comment:
not crazy about the name. I think the proper adjective here would be "exponentiated" but thats pretty clunky. maybe just `PowerProfile`? That would fit with `SumProfile` and `ProductProfile`

Reply:
`PowerProfile` I could see being mixed up with `PowerSeriesProfile` actually, maybe `ExponentProfile`?

Reply:
`RaiseProfile`

Reply:
I decided that `PowerProfile` is the least bad of all the options